seed/main/kgp.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE ldml SYSTEM "../../common/dtd/ldml.dtd">
+<!-- Copyright © 1991-2020 Unicode, Inc.
+For terms of use, see http://www.unicode.org/copyright.html
+Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
+CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
+
+Proper interpretation of these files requires synthesis of missing items, as per
+​http://unicode.org/repos/cldr/trunk/specs/ldml/tr35-general.html#Annotations
+Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/ for derived annotations.
+-->
+<ldml>
+	<identity>
+		<version number="$Revision$"/>
+		<language type="kgp"/>
+	</identity>
+	<localeDisplayNames>
+		<languages>
+			<language type="kgp">Kanhgág</language>
```

Comment:
I just want to make sure, do you have a source on that you should write language names with an initial-uppercased letter in this language? A common mistake when looking at the English file as reference is to think that language names should be in initial-uppercase in all translations (as well for months and weekdays). Wikipedia has the native language name written fully in lowercase: [Kaingang language](https://en.wikipedia.org/wiki/Kaingang_language), so either this file or the article needs updating.

Reply:
Once you post the comment here, I've sent the question to Kaingang linguists. Today I've received a feedback, and it was explained to me it was a practice in the past, but not rule, where they've used capital letter for languages as a way of show importance/relevance for the language. In addition, he confirmed the rule should be the same as Portuguese (lowercase). I'll update the PR right now.